<?xml version="1.0" encoding="UTF-8"?><testrun name="SbbApplicationTests" project="sbb" tests="1" started="1" failures="1" errors="0" ignored="0">
  <testsuite name="com.myste.sbb.SbbApplicationTests" time="0.46" displayname="SbbApplicationTests" uniqueid="[engine:junit-jupiter]/[class:com.myste.sbb.SbbApplicationTests]">
    <testcase name="testJpa" classname="com.myste.sbb.SbbApplicationTests" time="0.46" displayname="testJpa()" uniqueid="[engine:junit-jupiter]/[class:com.myste.sbb.SbbApplicationTests]/[method:testJpa()]">
      <failure>
        <expected>1</expected>
        <actual>2</actual>
        org.opentest4j.AssertionFailedError: expected: &lt;1&gt; but was: &lt;2&gt;&#13;
	at org.junit.jupiter.api.AssertionFailureBuilder.build(AssertionFailureBuilder.java:151)&#13;
	at org.junit.jupiter.api.AssertionFailureBuilder.buildAndThrow(AssertionFailureBuilder.java:132)&#13;
	at org.junit.jupiter.api.AssertEquals.failNotEqual(AssertEquals.java:197)&#13;
	at org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:150)&#13;
	at org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:145)&#13;
	at org.junit.jupiter.api.Assertions.assertEquals(Assertions.java:531)&#13;
	at com.myste.sbb.SbbApplicationTests.testJpa(SbbApplicationTests.java:153)&#13;
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)&#13;
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)&#13;
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)&#13;
&#13;

      </failure>
    </testcase>
  </testsuite>
</testrun>
